_pages/events/winter-school.md

```diff
@@ -10,12 +10,25 @@
 
 <div class="jumbotron" markdown="1">
 
-Winter School 2020 will be January 7, 2020 - February 2, 2020.
-
-The two mandatory lectures are Tuesday, January 7 and Thursday, January 9 from 19:00 - 21:00 (7pm - 9pm) in [26-100](https://whereis.mit.edu?q=26-100).
+Winter School 2021 will happen - virtually - from January 7, 2021 to January 28, 2021. **[Sign up here!](https://mitoc-trips.mit.edu/trips/1378/)**
 
 </div>
 
+### Winter school 2021
+
+Due to the current surge of cases locally and in accordance with [MIT's response to COVID-19](http://web.mit.edu/covid19/) and the various local travel orders, we will not be able to run any in-person event.
```

Comment:
"any in-person events"?

Reply:
Is it plural if there is none? Google seems to slightly prefer the singular

Reply:
:shrug: English is weird. Both are valid, but I think "not running any <plural noun>" is a bit more idiomatic.

As with most things grammatical, it doesn't matter much so long as folks understand the meaning. I think it's fine to leave as-is. :)